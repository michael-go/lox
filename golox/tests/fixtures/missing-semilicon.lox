// hello
// are you here?
print "here not here not here"