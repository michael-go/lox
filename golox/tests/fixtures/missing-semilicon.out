# exit code: 1
# stdout:

# stderr:
[line 3] Error at end: Expect ';' after value.
exit status 65

